frontend/src/config/config.js

```diff
@@ -2,7 +2,7 @@ const config = require("./data/config.yml");
 
 // Load server data with defaults if file doesn't exist
 try {
-    const servers = require("./data/servers.yml");
+    var servers = require("./data/servers.yml");
```

Comment:
The correct way I believe is to have a `var servers` declaration before the try and remove the `var` from everywhere else in the file